daemon/src/setup_contract.rs

```diff
@@ -241,6 +241,29 @@ pub async fn new(
     })
 }
 
+// Create a close transaction based on the current contract and a settlement proposals
+pub fn close_transaction(
```

Comment:
I don't think this should be in `contract_setup`. It has nothing to do with setting up a contract :)

I would suggest making this a function on `Dlc`.

Reply:
great idea.